Library/Homebrew/utils.rb

```diff
@@ -640,3 +640,13 @@ def number_readable(number)
   (numstr.size - 3).step(1, -3) { |i| numstr.insert(i, ",") }
   numstr
 end
+
+def homebrew_user_agent
+  parentheticals = [
+    "Ruby #{RUBY_VERSION}-#{RUBY_PATCHLEVEL}",
+    OS_VERSION,
+  ]
+  is_ci = ["CI", "JENKINS_URL"].any? { |var| ENV.include? var }
```

Comment:
I think an inline `ENV["CI"] || ENV["JENKINS_URL"]` is the preferable solution unless we need this test in more places.